### PR TITLE
Implement help button (show spec)

### PR DESCRIPTION
**Pull request by Siegrift**
Show a spec page on a new tab using cmd+h or clicking a help icon.

#33

### CODE REVIEW

File: src/common/Header.js [hunk 1]
```diff
@@ -53,6 +54,15 @@ const Header = ({ user, classes }) =>
           </Avatar>
         )}
 
+        <Button
+          className={classes.item}
+          variant="outlined"
```
Comment (mikx98) [inline src/common/Header.js]: maybe this: https://material-ui.com/demos/buttons/#floating-action-buttons would be nicer :thinking:

File: src/common/Header.js [hunk 2]
```diff
@@ -53,6 +54,15 @@ const Header = ({ user, classes }) =>
...
+          color="inherit"
+          href={SPEC_URL}
+          target="_blank"
```
Comment (mikx98) [inline src/common/Header.js]: :pray: https://medium.com/@ali.dev/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f

File: src/worklogs/WorkLogs.js [hunk 3]
```diff
@@ -82,6 +82,9 @@ export default compose(
       clearSelection,
       setJiraIssueForSelection
     }) => e => {
+      e.preventDefault()
+      e.stopPropagation()
```
Comment (mikx98) [inline src/worklogs/WorkLogs.js]: nope, we don't want this - you won't need it after rebase to master

File: src/worklogs/WorkLogs.js [hunk 4]
```diff
@@ -153,6 +156,11 @@ export default compose(
           }
           break
 
+        case 'h':
+          if (e.ctrlKey) {
+            window.open(SPEC_URL, '_blank')
```
Comment (mikx98) [inline src/worklogs/WorkLogs.js]: the security issue :top:

**Comment by mikx98** [comment]
Also, please mark that this PR closes the related issue

File: src/common/Header.js (hunk 1 above)
Comment (Siegrift) [reply src/common/Header.js]: I used the IconButton, I think it looks better now (FABs were not looking good)

File: src/common/Header.js (hunk 2 above)
Comment (Siegrift) [reply src/common/Header.js]: :+1:

**Comment by Siegrift** [comment]
I've changed the shortcut, because we are fighting with default browser actions (I've tested on chrome and firefox developer)

**Pull request merged**